edx_repo_tools/dependabot_yml.py

```diff
@@ -36,8 +41,22 @@ def _add_elements(self):
         if not found:
             self.elements['updates'].append(self.yml_instance.load(value))
 
+
```

Comment:
can you remove an extra line break from here, https://peps.python.org/pep-0008/#blank-lines

Reply:
done